### ISSUE TITLE
Outdated README instructions for serving break new workbench template

### ISSUE THREAD

**Issue by DanilaFe** [hpc-carpentry/hpc-chapel#24]
Hello! I'm trying to suggest some changes to bring the Chapel code up to date w.r.t. the latest release, and ran into trouble getting started. All is well now, but I thought I'd mention it here, and provide the error message in case someone else runs into the same issue.

The `README.md` file for this repo currently instructs running `make serve` to preview the lessons.  This results in installing some Jekyll gems and generating a `_site` folder in the repository. However, since this lesson has been updated to the new carpentry template in https://github.com/hpc-carpentry/hpc-chapel/pull/23, the Jekyll build does not give a "proper-looking" website (themes are missing and some markdown content is not correctly rendered).

Moreover, following the [standard setup process for the carpentries workbench](https://carpentries.github.io/sandpaper-docs/), running `sandpaper::build_lesson` in the folder causes the following error:

```
Error in UseMethod("xml_find_first") :
  no applicable method for 'xml_find_first' applied to an object of class "xml_document"
```

Deleting the Jekyll-generated `_site` fixed this problem (and `sandpaper::build_lesson` built everything just fine). However, perhaps the Makefiles and the README ought to be updated with the preview instructions?

**Comment by bkmgit** [comment]
Thanks for reporting this.  Pull requests are welcome!

**Comment by razoumov** [comment]
Hi Daniel - thank you for the suggestion. I created a pull request with a suggested change https://github.com/hpc-carpentry/hpc-chapel/pull/26 -- someone will need to review and hopefully approve it.

**Comment by razoumov** [comment]
@DanilaFe Ok, I suggested the PR 26 to address this, and it did not pass the PR checks: "Error: Failed to CreateArtifact: Received non-retryable error: Failed request: (409) Conflict: an artifact with this name already exists on the workflow run"

I don't know which file caused this error. I simplified this PR with a new PR 28 to replace it, and I get the same error. I modified only two files there: Makefile and README.md, so apparently I am not at liberty to edit those two files. Suggestions?
